composer.json

```diff
@@ -20,7 +20,7 @@
         }
     ],
     "require": {
-        "php": ">=7.1.3,<7.4",
```

Comment:
Can we keep `,<8.0`?

Reply:
Nvermind this.

Reply:
Should I do also that? Received an email with "nevermind" but probably you deleted your comment so I'm a little dazed with this

Reply:
Yeah, we should include it to prevent PHP 8. Once PHP 8 comes out we can confirm compatibility and allow it.

Reply:
I think is acceptable to use `^7.2` as done with last changes done

Reply:
Given there's no php 7.5, i agree we're good :)